Sources/Library/Parsing/XcodeWorkspace/XcodeWorkspace.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import SwiftPackage
+
+public struct XcodeWorkspace: Equatable {
+    public let name: String
+    public let xcodeProjectURL: URL
```

Comment:
xcworkspace may have multiple projects

Reply:
Ahh yes, I wanted to support that in a separate pull request.